cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -248,12 +260,16 @@ Basic Authentication
 The simplest way to identity a client is to authenticate against a realm file.
 To configure basic authentication add the following properties to ``cdap-site.xml``:
 
-========================================================== ==================================================================
-Property                                                   Value
-========================================================== ==================================================================
-``security.authentication.handlerClassName``               ``co.cask.cdap.security.server.BasicAuthenticationHandler``
-``security.authentication.basic.realmfile``                ``<path>`` *(either absolute or relative)*
-========================================================== ==================================================================
+========================================================== =========================== ======================================
+Property                                                   Default Value               Description
```

Comment:
These aren't "Default Values", these are the specific configurations needed to enable use of realm-file based authentication of users.